Sources/MapboxCoreNavigation/CoreNavigationNavigator.swift

```diff
@@ -45,6 +45,8 @@ class Navigator {
         return RoadObjectsStore(try! navigator.roadObjectStore())
     }()
     
+    static var accessToken: String? = nil
```

Comment:
Let’s store the DirectionsCredentials instead of the bare access token. That way the application can also customize the API endpoint.

Reply:
Thanks for the suggestion, I'll update this part.